test/fixtures/output/byok/create_1.golden

```diff
@@ -0,0 +1,29 @@
+Created self-managed key "cck-004".
+
+Copy and append these permissions into the key policy "Statements" field of the ARN in your AWS key management system to authorize access for your Confluent Cloud cluster.
+
+{
+	"Sid" : "Allow Confluent accounts to use the key",
+	"Effect" : "Allow",
+	"Principal" : {
+		"AWS" : [
+			"arn:aws:iam::123456789012:role/role1",
+			"arn:aws:iam::123456789012:role/role2"
+		]
+	},
+	"Action" : [ "kms:Encrypt", "kms:Decrypt", "kms:ReEncrypt*", "kms:GenerateDataKey*", "kms:DescribeKey" ],
+	"Resource" : "*"
+}, {
+	"Sid" : "Allow Confluent accounts to attach persistent resources",
+	"Effect" : "Allow",
+	"Principal" : {
+		"AWS" : [
+			"arn:aws:iam::123456789012:role/role1",
+			"arn:aws:iam::123456789012:role/role2"
+		]
+	},
+	"Action" : [ "kms:CreateGrant", "kms:ListGrants", "kms:RevokeGrant" ],
+	"Resource" : "*"
+}
+
+
```

Comment:
Super picky, but there should only be one newline at the end.

Reply:
Fixed.